agent/replacecall/src/test/java/randoop/instrument/ReplacementFileTest.java

```diff
@@ -19,7 +19,7 @@ public void missingClassMethodTest() throws IOException, ReplacementFileExceptio
     Path file = Paths.get("build/resources/test/randoop/instrument/missingclassreplacement.txt");
     thrown.expect(ReplacementFileException.class);
     String msg =
-        "missingclassreplacement.txt:2: Class not found: randoop.mock.Gamma for line: alpha.beta.Gamma.delta() randoop.mock.Gamma.delta(alpha.beta.Gamma)";
+        "missingclassreplacement.txt:2: Class not found: randoop.mock.Gamma for line: randoop.MethodReplacements.getAgentPath() randoop.mock.Gamma.delta(alpha.beta.Gamma)";
```

Comment:
It's perhaps out of scope for this pull request, but I would find this easier to read as
"Class randoop.mock.Gamma not found for line: ..."
and likewise for methods.